### PR TITLE
Disable base authentication for listed ip's

### PR DESCRIPTION
**Pull request by mdarii**
Some time we need to allow some host to access nagios server without auth.

### CODE REVIEW

**Comment by snkutlu** [comment]
@mdarii Seems OK. Needs some kitchen testing, anyways..  Then I can merge this..

File: attributes/default.rb [hunk 1]
```diff
@@ -225,3 +225,4 @@
 
 # attribute to add commands to source build
 default['nagios']['source']['add_build_commands'] = ['make install-exfoliation']
+default['nagios']['alloved_ips'] = []
```
Comment (snkutlu) [inline attributes/default.rb]: @mdarii I assume this is a typo, and should be "allowed_ips" :) Can you fix it here and below

Comment (mdarii) [inline attributes/default.rb]: @snkutlu Yes, sorry for that mistake, Already fixed it.

**Comment by mdarii** [comment]
kitchen tests added.

**Comment by mdarii** [comment]
@snkutlu hi, any update on that?

**Comment by mdarii** [comment]
Hi, any update on that?

**Comment by snkutlu** [comment]
@mdarii thx.. Sorry for late reply.. Merging now..

**Pull request merged**